libs/payments/ui/src/lib/client/components/CheckoutForm/index.tsx

```diff
@@ -260,6 +262,13 @@ export function CheckoutForm({
                   tagline: false,
                 }}
                 className="mt-6"
+                onError={() => {
+                  finalizeCartWithError(
```

Comment:
nit. Could this use `async/await` syntax instead? https://mozilla.github.io/ecosystem-platform/reference/style-guides/node-style-guide#asyncawait